components/counter-bar/src/index.js

```diff
@@ -162,13 +175,15 @@ CounterBar.Total.propTypes = {
   score: PropTypes.number,
   scoreBackgroundColor: PropTypes.string,
   scoreDisabledBackgroundColor: PropTypes.string,
+  tag: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
 };
 
 CounterBar.Total.defaultProps = {
   countBackgroundColor: WHITE,
   countDisabledBackgroundColor: BLACK,
   scoreBackgroundColor: RED,
   scoreDisabledBackgroundColor: GREY_3,
+  tag: 'a',
```

Comment:
to be honest, i'm not sure about the name of this prop

`component` may be a better name

Reply:
Yes, I was going to say this could be renamed to whatever you like. I'll change it to `component`.